### PR TITLE
Created recipe-hall and screening models

### PR DESCRIPTION
**Pull request by Minpeople**
1. I created recipe-hall model
2. I created screening model
3. Both models have relation between each other.
4. recipe-hall has name and capacity
5. Screening model belongs to recipe hall and it has own date (datetime)

### CODE REVIEW

File: db/migrate/20210804123238_create_screenings.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateScreenings < ActiveRecord::Migration[6.1]
+  def change
+    create_table :screenings do |t|
+      t.datetime :date
```
Comment (wmaciejak) [inline db/migrate/20210804123238_create_screenings.rb:6]: Consider that client may not want to have screenings without a date in his cinema ;) Consider the same in CinemaHalls migration

File: app/models/cinema_hall.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class CinemaHall < ApplicationRecord
+    has_many :screenings
```
Comment (jandudulski) [inline app/models/cinema_hall.rb:4]: It's a good practice to define `dependent` option on `has_many` relation, like: `has_many :screenings, dependent: :destroy` - the option value should depend on your needs. 

You can read more about the option (and possible values) in the [Rails documentation](https://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many).

File: db/migrate/20210804121256_create_cinema_halls.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateCinemaHalls < ActiveRecord::Migration[6.1]
+  def change
+    create_table :cinema_halls do |t|
+      t.string "name"
+      t.integer "capacity"
```
Comment (jandudulski) [inline db/migrate/20210804121256_create_cinema_halls.rb:7]: Are name and capacity optional?

File: test/fixtures/cinema_halls.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+two:
+  name: MyString
+  capacity: 1
+  rails: MyString
+  g: MyString
+  model: MyString
+  cinema_hall: MyString
+  name: MyString
+  capacity: 1
```
Comment (jandudulski) [inline test/fixtures/cinema_halls.yml:21]: Hm, it seems that something is duplicated here. E.g. `capacity` and `name`. And what is `g`?

**Review by jandudulski — APPROVED** [review]

**Pull request merged**